src/include/stir/recon_buildblock/BinNormalisationWithCalibration.h

```diff
@@ -0,0 +1,68 @@
+//
+//
+/*
+    Copyright (C) 2020, UCL
+    Copyright (C) 2020, NPL
+    See STIR/LICENSE.txt for details
+*/
+/*!
+  \file
+  \ingroup normalisation
+
+  \brief Declaration of class stir::BinNormalisationWithCalibration
+
+  \author Daniel Deidda
+  \author Kris Thielemans
+*/
+
+#ifndef __stir_recon_buildblock_BinNormalisationWithCalibration_H__
+#define __stir_recon_buildblock_BinNormalisationWithCalibration_H__
+
+
+#include "stir/RegisteredParsingObject.h"
+#include "stir/Bin.h"
+#include "stir/shared_ptr.h"
+#include "stir/recon_buildblock/BinNormalisation.h"
+#include "stir/decay_correction_factor.h"
+
+START_NAMESPACE_STIR
+
+/*!
+  \ingroup normalisation
```

Comment:
some documentation needed. say that when this class is used, normalisation data will also be calibrated. This means that a reconstructed image will have values that are calibrated as well.